src/quicklist.c

```diff
@@ -637,7 +637,7 @@ REDIS_STATIC int quicklistDelIndex(quicklist *quicklist, quicklistNode *node,
     }
     quicklist->count--;
     /* If we deleted the node, the original node is no longer valid */
-    return gone ? 1 : 0;
+    return gone;
```

Comment:
I see no point in this change, gone acts like a boolean and is set with either 1 or 0 above.

Reply:
i understood. there is my thought:
- conditional operator usually not easy to read (just in my opinion)
- get rid of one judgment (very very small ...)

@yoav-steinberg updated